dist/bm-angular-camera.js

```diff
@@ -130,12 +131,18 @@ function CameraController($scope, $element, Camera) {
     });
   };
 
+  vm.clearImage = function clearImage() {
+    if ($window.confirm(CLEAR_CONFIRM)) {
```

Comment:
For the demo sam asked for a javascript popup confirm, but that would be a nice feature. 

i'll make an issue for it so i can get on with other issues that need to be done for the demo